### PR TITLE
Add myself as maintainer

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
 
 requirements:
-  build:
-    - python <3
+  host:
+    - python >=3.6
     - pip 
   run:
-    - python <3
+    - python >=3.6
```
Comment (beckermr) [inline recipe/meta.yaml:22]: Does this work beyond importing on python 3? I cannot tell from the upstream repo, though I do see commits from 2019.

Comment (jan-janssen) [reply recipe/meta.yaml:22]: In the setup.py they seem to support both https://github.com/wikiotics/creoleparser/blob/master/setup.py#L37

Comment (beckermr) [reply recipe/meta.yaml:22]: Ah good call! I missed that bit. Thank you!

**Pull request merged**